tests/cypress/integration/actions_objects2/case_11_polylines_shape_track_label_points.js

```diff
@@ -63,8 +61,7 @@ context('Actions on polylines', () => {
             { x: 650, y: 200 },
             { x: 650, y: 250 },
         ],
-        complete: true,
```

Comment:
Let's rename a bit:

useDoneButton -> finishWithButton
complete -> finishWithHotkey

Reply:
Renamed. 
I have reworked it in such a way that there is no need to use ```finishWithHotkey``` explicitly since this action will be performed in the absence of parameters ```numberOfPoints``` and ```finishWithButton```